src/utils/moviesSlice.ts

```diff
@@ -0,0 +1,111 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export interface Movie {
```

Comment:
- define the interfaces as types if not extending it.
- define at the top but write the exports at the bottom of the file. ex: export  (Movie, TVShow)
- Also add a suffix as it is a type. ex: MovieListType, TVShowType. generally ask chatgpt for naming

Reply:
Resolved